modules/biglake-catalog/README.md

```diff
@@ -0,0 +1,77 @@
+# Biglake Catalog
+
+This module allows to create a Biglake catalog with databases and corresponding tables in each database.
```

Comment:
```suggestion
This module allows to create a BigLake Metastore with databases and corresponding tables in each database.
```
That's official naming according to the [docs](https://cloud.google.com/bigquery/docs/manage-open-source-metadata) - this also helps to understand what the module does (i.e. it doesn't provide ability to create external tables etc.)

Reply:
changed